lib/mutant/reporter/cli/printer/config.rb

```diff
@@ -11,13 +11,11 @@ class Config < self
           #
           # @return [undefined]
           #
-          # rubocop:disable AbcSize
-          #
           # @api private
           def run
             info 'Mutant configuration:'
             info 'Matcher:         %s',      object.matcher.inspect
-            info 'Integration:     %s',      object.integration.name
+            info 'Integration:     %s',      object.integration
             info 'Expect Coverage: %0.2f%%', (object.expected_coverage * 100)
```

Comment:
Do you know about the `%g` flag? It will strip trailing zeros from the decimal part.